src/stores/query.ts

```diff
@@ -6,20 +6,18 @@ import type { QueryItem, QueryValue } from "../types/queryData";
 import { writable, get } from "svelte/store";
 import { v4 as uuidv4 } from "uuid";
 import type { Category, Criteria } from "../types/catalogue";
+import { lensOptions } from "../stores/options";
 
 export const queryStore = writable<QueryItem[][]>([[]]);
 
-export const queryBase64Store = writable<string>("");
-
-/**
- * when the url has a query as base64 string, this will be parsed and the queryStore will be updated
- */
-const urlParams: URLSearchParams = new URLSearchParams(window.location.search);
-const queryParam: string | null = urlParams.get("query");
-
-if (queryParam !== null) {
-    const queryParamDecoded: QueryItem[][] = JSON.parse(atob(queryParam));
-    queryStore.set(queryParamDecoded);
+const encodedQuery = new URLSearchParams(window.location.search).get("query");
+if (encodedQuery !== null) {
+    const query = JSON.parse(
+        new TextDecoder().decode(
```

Comment:
If the String is not a Base64 String this will fail without any error message to the user. I would suggest a try and catch, if the string is not parseable, the query store should be set empty and a error should be shown.